crates/autopilot/src/infra/driver/mod.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Nomenclature wise, I think we should remove references to "driver" inside the autopilot. From the autopilot's perspective it is interfacing with "solvers" (the combination of driver/solver_engine)

Reply:
removed